src/main/java/ca/corefacility/bioinformatics/irida/service/impl/analysis/submission/AnalysisSubmissionServiceImpl.java

```diff
@@ -388,31 +383,16 @@ public AnalysisSubmission updatePriority(AnalysisSubmission submission, Analysis
 	 */
 	@Override
 	@PreAuthorize("hasRole('ROLE_USER')")
-	public AnalysisSubmission create(AnalysisSubmission analysisSubmission) throws ConstraintViolationException,
-			EntityExistsException {
-		UserDetails userDetails = (UserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+	public AnalysisSubmission create(AnalysisSubmission analysisSubmission)
+			throws ConstraintViolationException, EntityExistsException {
+		UserDetails userDetails = (UserDetails) SecurityContextHolder.getContext()
+				.getAuthentication()
+				.getPrincipal();
 		User user = userRepository.loadUserByUsername(userDetails.getUsername());
 		analysisSubmission.setSubmitter(user);
 
-		try {
-			return super.create(analysisSubmission);
-		} catch (final InvalidDataAccessApiUsageException e) {
-			// if the exception is because we're using unsaved properties, try to wrap the exception with a sane-er message.
-			if (e.getCause() != null) {
-				final Throwable primaryCause = e.getCause();
-				if (primaryCause.getCause() != null
-						&& primaryCause.getCause() instanceof TransientPropertyValueException) {
-					final TransientPropertyValueException propertyException = (TransientPropertyValueException) primaryCause
-							.getCause();
-					if (Objects.equals("namedParameters", propertyException.getPropertyName())) {
-						throw new UnsupportedOperationException(
-								"You must save the named properties *before* you use them in a submission.", e);
-					}
-				}
-			}
+		return super.create(analysisSubmission);
```

Comment:
I ditched the exception wrapping here.  We were just changing the `InvalidDataAccessApiUsageException` to a `UnsupportedOperationException`.   It was no longer getting thrown on `super.create(analysisSubmission)`, but instead when the transaction fired at the end of the method.  We weren't ever catching the exception anywhere except in the corresponding test so no issue with refactoring it out.